dev/openchpl_data-model.sql

```diff
@@ -2982,6 +2982,24 @@ CREATE TABLE openchpl.chpl_file
 		ON DELETE RESTRICT
 );
 
+CREATE TABLE IF NOT EXISTS openchpl.user_certification_body_map (
+	id bigserial NOT NULL,
+	user_id bigint NOT NULL,
+	certification_body_id bigint NOT NULL,
+	retired bool NOT NULL DEFAULT false,
```

Comment:
What is retired? Users can't be retired from an ACB, right?

Reply:
I asked this question on the API PR and Todd said that the relationship between user and ACB gets retired. I thought that was something new but just figured I missed it whenever we put that in.

Reply:
It does? When? Why?